README.md

```diff
@@ -61,7 +62,7 @@ _`npm init <initializer>` is available in npm 6+_
 yarn create react-app my-app
 ```
 
-_`yarn create` is available in Yarn 0.25+_
+_`yarn create <initializer>` is available in Yarn 0.25+_
```

Comment:
Can we link to the `yarn create` docs here?

Reply:
Sure!